packages/fxa-auth-db-mysql/scripts/read-security-events.js

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * This script takes the uid of a user and will print a security event object of that user
+ * from various tables like securityEvents, sessionTokens and devices tables from fxa database
+ */
+
+// Usage: node read-security-events.js uid
+
+const log = { trace: console.log, error: console.error, stat: console.log, info: console.log } //eslint-disable-line no-console
+const DB = require('../lib/db/mysql')(log, require('../db-server').errors)
+const config = require('../config')
+
+const securityEventsObject = {}
+const scriptInputUid = process.argv[2]
```

Comment:
Can you log an error message and exit if the uid is not specified? Otherwise this is displayed:

```bash
➜  scripts git:(farhan-gsoc) node read-security-events.js                                                                                                                                  git:(farhan-gsoc|)
buffer.js:207
    throw new ERR_INVALID_ARG_TYPE(
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The first argument must be one of type string, Buffer, ArrayBuffer, Array, or Array-like Object. Received type undefined
    at Function.from (buffer.js:207:11)
    at Object.<anonymous> (/Users/shanetomlinson/development/fxa/packages/fxa-auth-db-mysql/scripts/read-security-events.js:23:15)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:743:3)
```

Reply:
@shane-tomlinson done